internal/expr/parser.go

```diff
@@ -135,24 +144,73 @@ func (p *Parser) Parse(input string) (expr *ParsedExpr, err error) {
 			p.add(in)
 			continue
 		}
+	}
 
-		if bypass, ok, err := p.parseStringLiteral(); err != nil {
-			return nil, err
+	// Add any remaining unparsed string input to the parser.
+	p.partStart = p.pos
+	p.add(nil)
+	return &ParsedExpr{p.parts}, nil
+}
+
+// advance increments p.pos until it reaches content that might preceed a token
+// we want to parse.
+func (p *Parser) advance() error {
+
+	// The byte following these bytes might occur before the start of an expression.
+	delimiterBytes := map[byte]bool{
+		' ':  true,
+		'\t': true,
+		'\n': true,
+		'\r': true,
+		')':  true,
+		';':  true,
```

Comment:
Also a curious one. Where do we see this?  If it's splitting statements, we don't expect our variables to start a statement.

Reply:
True, that's not needed.